app/views/event_name.erb

```diff
@@ -0,0 +1,51 @@
+<div id="sources">
+  <div class='col-md-12 navbar navbar-default navbar-static-top navbar-inverse'>
```

Comment:
notice how you have this navbar with the movie and store on multiple pages - this isn't very DRY. we can use what's called a partial to help with this. a partial is basically a view template that's rendered within multiple other view templates. OR if you want this on every page, you could simply put it in the `layout.erb`